paddle/fluid/operators/math/cpu_vec.h

```diff
@@ -132,6 +132,121 @@ inline void vec_scal<float, platform::jit::avx512_common>(const int n,
   vec_scal<float, platform::jit::avx2>(n, a, x, y);
 }
 
+template <typename T, platform::jit::cpu_isa_t isa = platform::jit::isa_any>
+inline void vec_bias_sub(const int n, const T a, const T* x, T* y) {
```

Comment:
Can `vec_bias_sub` replaced with a more general name, it is a general math equation `Y = a - X` ? 
same with the below ones

Reply:
I though this name should be clear. 

Vector operation: one bias sub one vector then get other vector, corresponding to `vec_sub_bias` meaning `Y = X - a`.